src/test/java/org/jilt/test/data/defaultvalue/DefaultValueWithLombok.java

```diff
@@ -0,0 +1,39 @@
+package org.jilt.test.data.defaultvalue;
+
+import org.jilt.Builder;
+
+import java.util.Set;
+
+@Builder
+public class DefaultValueWithLombok {
+
+    @lombok.Builder.Default
```

Comment:
Let's add an import for `lombok.Builder.Default` instead of using the fully qualified name.